modules/data.atmosphere/R/met.process.R

```diff
@@ -227,24 +227,24 @@ met.process <- function(site, input_met, start_date, end_date, model,
       cf.id <- raw.id <- db.file
     }else{ 
       # I did this bc dbfile.input.check does not cover the between two time periods situation
-      mimetypeid <- get.id(table = "mimetypes", colnames = "type_string", 
+      mimetypeid <- PEcAn.DB::get.id(table = "mimetypes", colnames = "type_string", 
                            values = "application/x-netcdf", con = con)
 
-      formatid <- get.id(table = "formats", colnames = c("mimetype_id", "name"),
+      formatid <- PEcAn.DB::get.id(table = "formats", colnames = c("mimetype_id", "name"),
                          values = c(mimetypeid, "CF Meteorology"), con = con)
       
-      machine.id <- get.id(table = "machines", "hostname", PEcAn.remote::fqdn(), con)
+      machine.id <- PEcAn.DB::get.id(table = "machines", "hostname", PEcAn.remote::fqdn(), con)
       # Fidning the tiles.
       raw.tiles <- tbl(con, "inputs") %>%
         filter(
-          site_id == register$ParentSite,
+          .data$site_id == register$ParentSite,
           start_date >= start_date,
           end_date <= end_date,
```

Comment:
These lines are a bug! Both sides of the comparison are evaluated in the data mask, so they always compare equal.
```suggestion
          .data$start_date <= .env$start_date,
          .data$end_date >= .env$end_date,
```
Note the sign flip -- I think this step is looking for available inputs that cover the requested range, so I assume the original author was thinking of it as `.env$start_date >= .data$start_date`, so I reversed it to keep .data consistently on the left.